build/main.go

```diff
@@ -0,0 +1,113 @@
+package main
+
+import (
+	"context"
+	"os"
+
+	"dagger.io/dagger"
+	"golang.org/x/sync/errgroup"
+)
+
+var sema = make(chan struct{}, 5)
+
+func main() {
+	err := run()
+	if err != nil {
+		panic(err)
+	}
+}
+
+type buildFn func(context.Context, *dagger.Client, *dagger.Directory) error
+
+func run() error {
```

Comment:
Should we add a flag argument to select languages you wish to build as well?

Reply:
yeah can do that.

Reply:
will do in a follow up